src/github/repositories.ts

```diff
@@ -149,4 +149,29 @@ const terraformMigrationGuide = new github.Repository("terraform-migration-guide
     }
 );
 
-export const all = [infra, github_meta, awesome_pulumi, kubernetes_sdks, pulumi_concourse, pulumi_unifi, terraformMigrationGuide];
+const pulumi_astra = new github.Repository("pulumi-astra",
+    {
+        name: 'pulumi-astra',
+        description: 'Pulumi provider for datastax astra db',
+        hasDownloads: true,
+        hasIssues: true,
+        hasProjects: true, // false
+        hasWiki: true, // false
+        visibility: 'public',
+        vulnerabilityAlerts: true,
+        allowAutoMerge: false,
+        allowRebaseMerge: true,
+        allowSquashMerge: true, // false
+        allowMergeCommit: true,
+        deleteBranchOnMerge: false,
```

Comment:
I'd say we should make this t rue

Reply:
I'll take that along when standardizing the repo setups.